todo/todo/src/components/Filter.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+export const Filter = (props) => {
+    const {filter, setFilter} = props;
+    const filterChange = e => setFilter(e.target.value);
+    return (
+    <form>
+        <input type="radio"  onChange = {filterChange} value = "all" checked = {filter === 'all'} /><label>すべて</label>
```

Comment:
`=`の前後のスペースが空いている時とそうでない時でバラバラになっております。
タグの中の`=`は前後にスペースを空けないように記述するのが一般的です。
また、文字列を囲むときは`""`か`''`のどちらかで統一すると良いです。
他の箇所も同様に修正してみましょう。
```suggestion
        <input type="radio"  onChange={filterChange} value="all" checked={filter === "all"} /><label>すべて</label>
```